lib/swoosh/mailer.ex

```diff
@@ -53,7 +53,10 @@ defmodule Swoosh.Mailer do
       def deliver(%Swoosh.Email{html_body: nil, text_body: nil}) do
         raise ArgumentError, "expected \"html_body\" or \"text_body\" to be set"
       end
-      def deliver(%Swoosh.Email{} = email), do: @adapter.deliver(email, @config)
+      def deliver(%Swoosh.Email{} = email) do
```

Comment:
I multilined it because it felt like this function is the important one and I find it more readable